src/main/java/com/squareup/phrase/Phrase.java

```diff
@@ -41,7 +42,6 @@
  * </ul>
  * The constructor parses the original pattern into a doubly-linked list of {@link Token}s.
  * These tokens do not modify the original pattern, thus preserving any spans.
- * <p/>
```

Comment:
Leave in, otherwise javadoc will lump into one paragraph.

Also, you need to correct line 39 about valid keys.